### ISSUE TITLE
Strange errors in 1.10.2

### ISSUE THREAD

**Issue by NicolasPelletier** [testem/testem#937]
When I run `testem --debug` ( not `ci` mode ), I get the following error displayed on the last line of the UI of testem:
```
at processImmediate [as _immediateCallback] (timers.js:529:5)CollaborationServer/node_modules/testem/node_modules/bluebird/js/release/async.js:17:14)):504:31)
```

I looked into the log and I see nothing wrong.

I tried running the test again by pressing 'return' and it does not seem to add anything other than the following two lines in the log:
```
info Got keyboard Start Tests command 
info Triggered manually by pressing enter triggered test run. 
```

Once, I got the following error trace printed over the UI. In case it helps:
```
Unhandled rejection TypeError: runner.stop is not a function
   at <__ProjectFolder__>/node_modules/testem/lib/app.js:414:21
   at tryCatcher (<__ProjectFolder__>/node_modules/testem/node_modules/bluebird/js/release/util.js:16:23)━━━━━━━━━━━
   at Object.gotValue (<__ProjectFolder__>/node_modules/testem/node_modules/bluebird/js/release/reduce.js:145:18)
   at Object.gotAccum (<__ProjectFolder__>/node_modules/testem/node_modules/bluebird/js/release/reduce.js:134:25)
   at Object.tryCatcher (<__ProjectFolder__>/node_modules/testem/node_modules/bluebird/js/release/util.js:16:23)
   at Promise._settlePromiseFromHandler (<__ProjectFolder__>/node_modules/testem/node_modules/bluebird/js/release/promise.js:504:31)
   at Promise._settlePromise (<__ProjectFolder__>/node_modules/testem/node_modules/bluebird/js/release/promise.js:561:18)
   at Promise._settlePromiseCtx (<__ProjectFolder__>/node_modules/testem/node_modules/bluebird/js/release/promise.js:598:10)
   at Async._drainQueue (<__ProjectFolder__>/node_modules/testem/node_modules/bluebird/js/release/async.js:143:12)
   at Async._drainQueues (<__ProjectFolder__>/node_modules/testem/node_modules/bluebird/js/release/async.js:148:10)
   at Immediate.Async.drainQueues (<__ProjectFolder__>/node_modules/testem/node_modules/bluebird/js/release/async.js:17:14)
   at runCallback (timers.js:570:20)
   at tryOnImmediate (timers.js:550:5)
   at processImmediate [as _immediateCallback] (timers.js:529:5)
```

I am running 
 - TestEm 1.10.2
 - Mac OSX 10.11.6
 - Node 6.3.1
 - Npm 3.10.3

**Comment by alcinovanrooyen** [comment]
+1

**Comment by johanneswuerbach** [comment]
Could you share your testem config?

**Comment by NicolasPelletier** [comment]
@johanneswuerbach : I created a repo to demonstrate the bug: https://github.com/NicolasPelletier/Testem_Issue_937.

I cloned it, `npm i` and then `npm run test:dev` and BAM ! the problem happens. It may be that you need to press return to activate the bug, but 2 out of 3 times, it does the long trace I listed above.

**Comment by NicolasPelletier** [comment]
@johanneswuerbach : This afternoon I was on:
 - Node 4.4.2
 - npm 2.15.0

**Comment by JenHe** [comment]
I also ran into this issue on windows system. 

I am running on:
TestEm 1.10.1
Win 7 Enterprise with SP1
Node 4.4.7
Npm 2.15.8


Unhandled rejection TypeError: runner.stop is not a function
    at <__ProjectFolder__>\node_modules\testem\lib\app.js:414:21
    at tryCatcher (<__ProjectFolder__>\node_modules\testem\node_modules\bluebird\js\release\util.js:16:23)
    at Object.gotValue (<__ProjectFolder__>\node_modules\testem\node_modules\bluebird\js\release\reduce.js:145:18)
    at Object.gotAccum (<__ProjectFolder__>\node_modules\testem\node_modules\bluebird\js\release\reduce.js:134:25)
    at Object.tryCatcher (<__ProjectFolder__>\node_modules\testem\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (<__ProjectFolder__>\node_modules\testem\node_modules\bluebird\js\release\promise.js:504:31)
    at Promise._settlePromise (<__ProjectFolder__>\node_modules\testem\node_modules\bluebird\js\release\promise.js:561:18)
    at Promise._settlePromiseCtx (<__ProjectFolder__>\node_modules\testem\node_modules\bluebird\js\release\promise.js:598:10)
    at Async._drainQueue (<__ProjectFolder__>\node_modules\testem\node_modules\bluebird\js\release\async.js:143:12)
    at Async._drainQueues (<__ProjectFolder__>\node_modules\testem\node_modules\bluebird\js\release\async.js:148:10)
    at Immediate.Async.drainQueues [as _onImmediate] (<__ProjectFolder__>\node_modules\testem\node_modules\bluebird\js\release\async.js:17:14)
    at processImmediate [as _immediateCallback] (timers.js:383:17)

**Comment by johanneswuerbach** [comment]
Thanks for the detailed bug report, fix released in `1.10.3`.

**Comment by NicolasPelletier** [comment]
@johanneswuerbach : Thank you very much. It works for me now.
